python/unityagents/communicator.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from communicator_objects import UnityRLOutput, UnityRLInput,\
+    UnityOutput, UnityInput, AcademyParameters,\
+    UnityInitializationInput, UnityInitializationOutput
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("unityagents")
+
+
+class Communicator(object):
```

Comment:
Class comment

Reply:
Comment is in the __init__ method. Probably not the best place for it but it is consistent with the other classes of unityagents

Reply:
Consistency is great, but the class docstring shows up when users run help on the class...
It is also required by pep8: https://www.python.org/dev/peps/pep-0008/#documentation-strings

Reply:
There is one comment in the __init__ method, where should the class comment live ? Between 
```
class Communicator(object):
```
and 
```
    def __init__(self, worker_id=0,
                 base_port=5005):
```
?